src/com/google/enterprise/adaptor/opentext/OpentextAdaptor.java

```diff
@@ -935,9 +1043,11 @@ int getXmlSearchCount(Element documentElement) {
   @VisibleForTesting
   Element getXmlSearchResults(String authenticationToken, String query)
       throws IOException {
-    URL url = new URL(this.contentServerUrl + "?" + query);
+    URL url = new URL(this.indexingContentServerUrl + "?" + query);
     URLConnection conn = url.openConnection();
     conn.addRequestProperty(COOKIE, "LLCookie=" + authenticationToken);
+    conn.setConnectTimeout(this.connectTimeout);
```

Comment:
Is this continual use of "this." your own preference, or did it have another source?